src/app/components/client/DataBrokerProfiles.tsx

```diff
@@ -63,23 +71,14 @@ export const DataBrokerProfileCard = (props: DataBrokerProfileCardProps) => {
 
   return (
     <div className={styles.dataBrokerProfileCard}>
-      <div
-        data-broker={props.data.data_broker}
-        className={styles.imagePlaceholder}
-      />
-      {/* TODO: Add logic to show unique image per data broker */}
-      {/* <Image src={} alt={props.data.data_broker} /> */}
+      <span className={styles.dataBrokerName}>
+        {getDataBrokerName(props.data.data_broker)}
+      </span>
+
       <a href={props.data.link} target="_blank">
         {l10n.getString(
           "fix-flow-data-broker-profiles-view-data-broker-profiles-view-profile",
         )}
-        <span className={styles.openInNewTab}>
-          <OpenInNew
-            alt={l10n.getString("open-in-new-tab-alt")}
-            width="13"
-            height="13"
-          />
-        </span>
```

Comment:
I think these were initially forgotten in the designs (and are therefore probably not present in the Figma you received), but we did intentionally add these icons back in because the links open in a new tab.

Reply:
Added back 7d8349d5d22411f9bafd2b373b2ce0ca6ae03140